src/user/organization/organization.js

```diff
@@ -53,121 +54,182 @@ class Organization extends Component {
       description,
       // contactInfo
     } = orgProfile;
+
+    const handleClick = (type) => {
+      this.setState({ type: type })
+    }
+
+    const aboutContent = (
+      <Card className="about-us">
+        <Card.Body>
+          <div className="title">{orgProfile?.name}</div>
+          <div className="subtitle">Short description </div>
+          <p className="short__desc">
+            {description?.shortDescription ||
+              "Short details of organization"}
+          </p>
+          <div className="subtitle">About us in details</div>
+          <p className="long__desc">
+            {description?.longDescription ||
+              "Long description of the organization"}
+          </p>
+        </Card.Body>
+      </Card>
+    )
+
+    const contactContent = (
+      <>
+        {this.state.isLoading ? (
+          contactLoading()
+        ) : (
+          <OrgContact
+            admins={orginfo.admins}
+            website={orginfo.website}
+            contactinfo={orginfo.contactinfo}
+          />
+        )}
+      </>
+    )
+
+    const updatesContent = (
+      <>
+        {this.state.isLoading ? (
+          <div className="orgupdatesloading">{orgUpdatesLoading()}</div>
+        ) : (
+          <div className="org-updates">
+            <Updates></Updates>
+          </div>
+        )}
+      </>
+    )
+
+    let content;
+    if (type === "About") {
+      content = aboutContent;
+    }
+    if (type === "Updates") {
+      content = updatesContent;
+    }
+    if (type === "Contact") {
+      content = contactContent;
+    }
```

Comment:
Switch

Reply:
It was already there in Dashboard content! I was focusing on responsiveness.
I'll rectify in some other PR as similar things being used at multiple places.